readme.md

```diff
@@ -73,6 +73,7 @@ The plugin can be used in plug-n-play mode without any additional configuration.
 |-------------|---------------|----------------------------------------------------------------------------|----------|-----------------|
 | backup-dir  | string        | Directory to backup the original functions in (relative to base directory) | No       | ''              |
 | debug       | boolean       | Enables verbose logging for the plugin                                     | No       | false           |
+| quiet       | boolean       | Disable all logging for the plugin                                         | No       | false           |
```

Comment:
Could you please arrange these config options alphabetically? So `quiet` should go at the end. Same for the `toml` example below this table.